### PR TITLE
Updating README

### PR DESCRIPTION
**Pull request by FranDepascuali**
## Summary ##
**Trello card**: https://trello.com/c/Q0gIdK9j/3-agregar-readme

Updates README with a description of the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,34 @@
...
+
+## Configuration
+
+## Reporting Issues
+
+## Contributing
+
+## Maintainers
+
+## Third party libraries
```
Comment (guidomb) [inline README.md]: I woudn't put this. Just read the Cartfile.

**Comment by guidomb** [comment]
What happened with this one?

**Comment by FranDepascuali** [comment]
@guidomb sorry, I'm continuing with this

**Comment by woloxci** [comment]
:+1: There are no linter violations.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,82 @@
...
+  * [Installation](#installation)
+  	* [Carthage](#carthage)
+  	* [Manually](#manually)
+  * [Usage](#usage)
+  * [Bootstrap](#bootstrap) 
+  * [Contributing](#usage)
+  * [About](#about)
+  * [License](#license)
+
+## Bootstrap
```
Comment (danielaRiesgo) [inline README.md]: I would move this after Usage, following the table of contents' order.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,82 @@
...
 git clone git@github.com:Wolox/wolmo-core-ios.git
 cd wolmo-core-ios
 script/bootstrap
 ```
+
+## Installation
+
+### Carthage
+
+Carthage is a dependency manager that builds your dependencies and provides you with binary frameworks.
```
Comment (danielaRiesgo) [inline README.md]: I would add a link to carthage repo or documentation.

File: README.md [hunk 4]
```diff
@@ -1,12 +1,82 @@
...
+To download wolmo-core-iOS, add this to your Cartfile:
+```
+github "Wolox/wolmo-core-ios" "master"
+```
+
+### Manually
+[Bootstrap](#bootstrap) the project and then drag it to your workspace.
+
+## Contributing
+1. [Bootstrap](#bootstrap)
```
Comment (danielaRiesgo) [inline README.md]: Shouldn't they fork?

File: README.md [hunk 5]
```diff
@@ -1,12 +1,82 @@
...
+brew install carthage
+```
+To download wolmo-core-iOS, add this to your Cartfile:
+```
+github "Wolox/wolmo-core-ios" "master"
+```
+
+### Manually
+[Bootstrap](#bootstrap) the project and then drag it to your workspace.
+
```
Comment (danielaRiesgo) [inline README.md]: Usage is missing. Is it for a future PR?

File: README.md [hunk 6]
```diff
@@ -44,13 +36,21 @@ github "Wolox/wolmo-core-ios" "master"
...
-1. [Bootstrap](#bootstrap)
-2. Create your feature branch (`git checkout -b my-new-feature`)
-3. Commit your changes (`git commit -am 'Add some feature'`)
-4. Run tests (`./script/test`)
-7. Push your branch (`git push origin my-new-feature`)
-8. Create a new Pull Request
+1. Fork it
+2. [Bootstrap](#bootstrap)
+3. Create your feature branch (`git checkout -b my-new-feature`)
+4. Commit your changes (`git commit -am 'Add some feature'`)
```
Comment (danielaRiesgo) [inline README.md]: `-am`: I don't know if its a typo or if it exists (because I just use `-m`)

File: README.md [hunk 7]
```diff
@@ -44,13 +36,21 @@ github "Wolox/wolmo-core-ios" "master"
...
+
 ## Contributing
-1. [Bootstrap](#bootstrap)
-2. Create your feature branch (`git checkout -b my-new-feature`)
-3. Commit your changes (`git commit -am 'Add some feature'`)
-4. Run tests (`./script/test`)
-7. Push your branch (`git push origin my-new-feature`)
-8. Create a new Pull Request
+1. Fork it
+2. [Bootstrap](#bootstrap)
```
Comment (danielaRiesgo) [inline README.md]: I would add something like:  bootstrap the forked repository, that would be the same except that in the cloning, `Wolox` would be replaced by your user.

File: README.md [hunk 8]
```diff
@@ -44,13 +36,21 @@ github "Wolox/wolmo-core-ios" "master"
...
+
 ## Contributing
-1. [Bootstrap](#bootstrap)
-2. Create your feature branch (`git checkout -b my-new-feature`)
-3. Commit your changes (`git commit -am 'Add some feature'`)
-4. Run tests (`./script/test`)
-7. Push your branch (`git push origin my-new-feature`)
-8. Create a new Pull Request
+1. Fork it
+2. [Bootstrap](#bootstrap) using the forked repository (instead of `Wolox/wolmo-core-ios.git`, `clone your-user/wolmo-core-ios.git`)
```
Comment (danielaRiesgo) [inline README.md]: I would write:
(instead of `Wolox/wolmo-core-ios.git`, `your-user/wolmo-core-ios.git`)
without `clone`.

**Comment by danielaRiesgo** [comment]
👍

**Comment by guidomb** [comment]
👍

**Pull request merged**